internal/cri/server/sandbox_remove.go

```diff
@@ -86,6 +86,11 @@ func (c *criService) RemovePodSandbox(ctx context.Context, r *runtime.RemovePodS
 		return nil, fmt.Errorf("failed to delete sandbox %q: %w", id, err)
 	}
 
+	err = c.cleanupImageMounts(ctx, id)
```

Comment:
tough call on whether the clean up should be here or above the ShutdownSandbox call.. as if the clean up fails we'll have a store entry for the cri pod that is no longer in sandboxService.. not huge since we removed all the containers still..

@mxpv wdyt?

Reply:
yeah let's move this to before the sandbox shutdown otherwise there would be no way to try again..